src/translations/fr/campaign.php

```diff
@@ -406,7 +406,7 @@
     'No sendouts have been sent to this mailing list.' => 'Aucune campagne n’a été envoyée à cette liste de diffusion.',
     'No synced mailing lists exist.' => '',
     'No user groups exist.' => 'Aucun groupe d’utilisateurs existant.',
-    'None' => 'Auncun',
+    'none' => 'Non inscrit',
```

Comment:
Please restore the key to be capitalised (`None`) as translations are case-sensitive.